pkg/logging/logging.go

```diff
@@ -130,6 +133,7 @@ func LogStructure(argName string, argStruct interface{}, log *zerolog.Logger, fo
 	if err != nil {
 		log.Warn().Msg("Failed converting " + argName + " to json")
```

Comment:
in order to prevent missing messages, maybe to add 
log.WithLevel(verbosity).CallerSkipFrame(1).Bool(FORUSER, forUser).Bool(AUDIT, audit).Msg(argName + ": " + fmt.Sprintf("%v", argStruct))

Reply:
done